config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -509,7 +509,7 @@ periodics:
           - '--node-test-args=--feature-gates="NodeSwap=true" --service-feature-gates="NodeSwap=true" --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --fail-swap-on=false --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd*\"]}"'
           - --node-tests=true
           - --provider=gce
-          - --test_args=--nodes=1 --focus="NodeSwap" --skip="\[Flaky\]|\[Slow\]|\[Benchmark\]|\[Feature:Eviction\]"
+          - --test_args=--nodes=1 --focus="\[Feature:NodeSwap\]|\[Feature:Eviction\]" --skip="\[Flaky\]|\[Benchmark\]"
```

Comment:
Again the main issue with this is that you will get a failingImageGC test and you will bring long running tests into this pipeline. We may hit timeouts included all the jobs and we don’t need to test disk based or pid eviction with swap.

We either bring this in the serial lane and filter to MemoryEvictions.

Or we can add a decided eviction job with swap if you don’t want to filter. I still prefer that we only run the memory evictions tests tests on a swap enabled node as those are the only ones that make sense for Swap.

Reply:
@kannon92 Done. PTAL.